notes/push/page.sh

```diff
@@ -0,0 +1,292 @@
+#| | b0d5f11c-f77d-4e68-82d5-1b371e01916b | dba0c729d0d33f684d1f77175532943aa5fce91e | 820b236a-0fc1-42f3-8aae-b5181af72e42 | 2023-09-15 11:54:47.733171 | 0     |      | 6       | Alytus      |
+#| | 383c5c3d-1767-4fbd-8bcc-7819423a6049 | 73a19c86dffa9363cbaf9f2bd9d1d5fbbe354bc0 | d86cc411-726d-4897-bd7a-b59d9e7d09d3 | 2023-09-15 11:54:47.733456 | 0     |      | 7       | Marijampolė |
+#| | ac1e4d24-b6ff-4f63-8cc3-727760563941 | 0989120804b58b747ff998744808f8d3c8188534 | 86b3e49a-d882-4ce9-9487-3cef05135666 | 2023-09-15 11:54:47.733729 | 0     |      | 8       | Mažeikiai   |
+#| | 374b862f-d01e-42ac-bcad-0ca8f97aff15 | c64c87682eb80327e3c03ff5f36827abc3151254 | acbc66fc-a26d-41ad-a172-17daf8b4ae25 | 2023-09-15 11:54:47.734023 | 0     |      | 9       | Jonava      |
+#| | 77327c99-6c06-4476-a0a5-4ef3ff0d1fb7 | 2e74a59cb754fe07503c51bbaf4c774cea6bf9a0 | eb87e26a-5bd9-48de-af99-e8183f7400b4 | 2023-09-15 11:54:47.734257 | 0     |      | 10      | Utena       |
+#| +--------------------------------------+------------------------------------------+--------------------------------------+----------------------------+-------+------+---------+-------------+
+
+http GET "$SERVER/$DATASET/City?format(ascii)"
+#| _page                                                                                     id  name      
+#| ---------------------------------------------------------------------------------------   --  ----------
+#| WzEsICJWaWxuaXVzIiwgImRjZjI5MWNlLTgzNWQtNDUxNy04NWNiLTBlZjAyZTJiYmEwYyJd                  1   Vilnius
+#| WzIsICJLYXVuYXMiLCAiMDJiMWI0MmEtZjJhZS00NTg3LThkMTAtMDdjMzgxOTQ0MDkzIl0=                  2   Kaunas
+#| WzMsICJLbGFpcFx1MDExN2RhIiwgImFhZTZmMWJhLTU3NjgtNGUyNi04MTBjLTQwMDE4NmNhY2U1NyJd          3   Klaipėda
+#| WzQsICJcdTAxNjBpYXVsaWFpIiwgIjQ5ZmM5OTljLWNkOTctNDJhZS1hMzkwLWI2ZmVhNDMyYWY1MCJd          4   Šiauliai
+#| WzUsICJQYW5ldlx1MDExN1x1MDE3ZXlzIiwgIjMzMWQxZjJjLTVhNDMtNDk2ZS1hNTAzLWQ2OTAzYTQzMmQ5ZiJd  5   Panevėžys
+#| WzYsICJBbHl0dXMiLCAiYjBkNWYxMWMtZjc3ZC00ZTY4LTgyZDUtMWIzNzFlMDE5MTZiIl0=                  6   Alytus
+#| WzcsICJNYXJpamFtcG9sXHUwMTE3IiwgIjM4M2M1YzNkLTE3NjctNGZiZC04YmNjLTc4MTk0MjNhNjA0OSJd      7   Marijampolė
+#| WzgsICJNYVx1MDE3ZWVpa2lhaSIsICJhYzFlNGQyNC1iNmZmLTRmNjMtOGNjMy03Mjc3NjA1NjM5NDEiXQ==      8   Mažeikiai
+#| WzksICJKb25hdmEiLCAiMzc0Yjg2MmYtZDAxZS00MmFjLWJjYWQtMGNhOGY5N2FmZjE1Il0=                  9   Jonava
+#| WzEwLCAiVXRlbmEiLCAiNzczMjdjOTktNmMwNi00NDc2LWEwYTUtNGVmM2ZmMGQxZmI3Il0=                  10  Utena
+
+
+echo "WzEsICJWaWxuaXVzIiwgImRjZjI5MWNlLTgzNWQtNDUxNy04NWNiLTBlZjAyZTJiYmEwYyJd" | base64 --decode
+#| [1, "Vilnius", "dcf291ce-835d-4517-85cb-0ef02e2bba0c"]
+# XXX: Here I run Spinta with internal mode, whre page key should always be
+#      _id, but here I get more than _id. Shouldn't this be change to use only
+#      _id?
```

Comment:
Shouldn't internal backend use only _id as page key, if sort is not used?

Reply:
it is done this way, so that internal backend would still abide by given manifest page()